docs/conf.py

```diff
@@ -73,7 +73,7 @@
     ),
     "django": (
         "http://docs.djangoproject.com/en/4.2/",
-        "http://docs.djangoproject.com/en/4.2/_objects/",
+        None,
```

Comment:
Why do we remove that and why don't we just remove it and we put `None` instead?

Reply:
I saw a warning in CI that the URL has moved, and it apparently moved back to the default Sphinx location, which is signalled with the `None` value. See the other intersphinx mappings above that have the same pattern :)